<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>staff_manage</class>
 <widget class="QDialog" name="staff_manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>教师管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#staff_manage{
background-color: rgb(255, 255, 255);
border-image: url(:/image/image/backpage_common.png);
}</string>
  </property>
  <widget class="QComboBox" name="cbb_age">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/image/image/icon/arrow_down.png);
}


QComboBox#cbb_age{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_age:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
   </property>
   <item>
    <property name="text">
     <string>年龄</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>31</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>33</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>34</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>35</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>36</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>37</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>39</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>41</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>42</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>43</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>44</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>45</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>46</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>47</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>48</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>49</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>51</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>52</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>53</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>54</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>55</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>56</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>58</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>59</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>61</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>62</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>63</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>65</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255, 255, 255,128);</string>
   </property>
   <property name="placeholderText">
    <string>姓名</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255, 255, 255,128);</string>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="placeholderText">
    <string>工号</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_faculty">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/image/image/icon/arrow_down.png);
}


QComboBox#cbb_faculty{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_faculty:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
   </property>
   <item>
    <property name="text">
     <string>学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地球科学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>资源学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>材料与化学学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>环境学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地球物理与空间信息学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海洋学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>机械与电子信息学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>自动化学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>经济管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>外国语学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>地理与信息工程学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数学与物理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>珠宝学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公共管理学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>计算机学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>体育学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>艺术与传媒学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>马克思主义学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>李四光学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高等教育研究所</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>远程与继续教育学院</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>国际教育学院</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button_add">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_add{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#button_add:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#button_add:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#button_add:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_delete">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_delete{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#button_delete:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#button_delete:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#button_delete:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255, 255, 255,128);</string>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rbt_male">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>男</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">sexGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbt_female">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>女</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">sexGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_revision">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#button_revision{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#button_revision:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#button_revision:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#button_revision:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sexGroup"/>
 </buttongroups>
</ui>
